e2e/atlas/integrations_test.go

```diff
@@ -50,6 +50,19 @@ func TestIntegrations(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
+
+	projectName := fmt.Sprintf("e2e-proj-%v", n)
```

Comment:
[nit] can we rename this, sometimes test left things around in atlas and this way we can better diagnose where are we leaking data
```suggestion
	projectName := fmt.Sprintf("e2e-integration-proj-%v", n)
```